src/oge/data_cleaning.py

```diff
@@ -697,6 +699,34 @@ def clean_cems(year: int, small: bool, primary_fuel_table, subplant_emission_fac
     # load the CEMS data
     cems = load_data.load_cems_data(year)
 
+    for column in ["co2_mass_lb", "nox_mass_lb", "so2_mass_lb"]:
```

Comment:
We should probably rename `bad_pollutant_data` to be more specific like `negative_emissions_data` if we're just testing for negative values. And maybe add an inline comment at the beginning like "treat negative emissions as bad data and replace with missing values.

We might want to make this a separate function in `validation` rather than just implement this inline? What do you think? Or we could just keep it here

Reply:
I don't have any preference. It is an edge case that happen only in cems so we could leave the logic here. We do the same thing in the `clean_eia923` function for `fuel_consumed_mmbtu` and `fuel_consumed_for_electricity_mmbtu`

Reply:
Added comments and renamed variable but I did not create a separate function